elasticgraph-graphql/spec/support/sub_aggregation_support.rb

```diff
@@ -16,12 +16,20 @@ module Aggregation
           {"size" => size, "adapter" => grouping_adapter.meta_name}.merge(hash)
         end
 
-        def inner_terms_meta(hash = {})
-          {"merge_into_bucket" => {}}.merge(hash)
+        def inner_terms_meta(grouping_fields, hash = {})
+          {
+            "grouping_fields" => grouping_fields.is_a?(Array) ? grouping_fields : [grouping_fields],
```

Comment:
```suggestion
            "grouping_fields" => Array(grouping_fields),
```

That'll coerce to an array for you.